README.md

```diff
@@ -1,15 +1,7 @@
 # FEC-Data-Wranglin
-get_that_data.py pulls data from FEC API to use later on ML project. (ETA? IDK)
-
-If you want to make more than a handful of requests you need an api_key, visit https://api.open.fec.gov/developers/#/ to get a key.
-
-You will break the program if you put anything other than a key or hitting enter for DEMO_KEY when asked for input. I'll fix it later.
-And yes I know the data processing is wildly inadequate, that's on my fix-it list too.
-
-Requires numpy and urllib.request
-
-If you want to test it without downloading all of the FEC's data, check the Special Note in the docstring for get_the_data()
+get_that_data.py pulls data from FEC API to use later on ML project. 
 
+ 
+If you want to make more than a handful of requests you need an api_key, visit https://api.open.fec.gov/developers/#/ to get a key. Set your api_key as an environment variable. i.e. FEC_API_KEY=DEMO_KEY
+		 
 Run python3 get_that_data.py in the terminal
```

Comment:
Was happy to see an update to this file, I was going to open up a tiny Issue soon for a couple of edits. Happier yet that there's less self-abasement this time around :)

Two notes on the README:
- wrapping `variables` and `commands` in backticks is half the fun of GitHub
- I'll defer to you if this is premature or not, but I think a small section explaining the resulting data would go a long way. Maybe a couple short sentences and a Markdown table of `data.head(5)`? [This guide should help](https://guides.github.com/features/mastering-markdown). Or if you're as lazy as me, you'll bookmark [this page](https://donatstudios.com/CsvToMarkdownTable), lol

Reply:
You say lazy I say time efficient, as soon as I saw what it did it was bookmarked.